### DISCUSSION TITLE
Server responded with an error

### DISCUSSION THREAD

**Discussion by larffxx** [TopiSenpai/LavaSrc#181] [Q&A] [answered]
After adding spotify playlist have an exception 
![image](https://github.com/topi314/LavaSrc/assets/76408500/cbf9ca40-82ff-4547-ac52-28eef6a41dad)

**Comment by topi314** [comment]
quite frankly I am not able to help you in any meaningful way with this little information
if you are making a bug report want help please include as much information as possible

what playlist triggered this?
are your api credentials valid?

ideally post your logs

**Reply by larffxx** [reply]
every playlist trying to add response this exception this work yesterday, but today it's cause by 
![image](https://github.com/topi314/LavaSrc/assets/76408500/32d14f34-0b0b-4425-9ebe-530da8c484d2)
sry for bothering

**Reply by larffxx** [reply, marked as answer]
enable vpn and it's work 
Sorry :(
